packages/examples/import/AMFImport/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "amf-import",
+  "version": "0.0.1",
+  "description": "Demonstrating AMF import. Drag the AMFImport folder onto the JSCAD instance.",
+  "main": "index.js",
+  "keywords": [
+    "jscad",
+    "cad"
+  ],
+  "author": "Simon Clark",
+  "license": "MIT",
+  "dependencies": {
+    "@jscad": "2.0.0"
```

Comment:
it will be interesting once V2 is finally published. but i don't think that the jscad project will be a dependency for designs. designs will have dependencies on other designs.